<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecordsDialog</class>
 <widget class="QDialog" name="AddRecordsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>693</width>
     <height>553</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="regionValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fixedPhoneValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fixedPhoneUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobilePhoneUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="yearLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>年份：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fixedPhoneLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>固定电话用户数：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobilePhoneValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="internetValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fixedAssetsValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="communicationValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="infoServiceValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="infoProductValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="internetUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fixedAssetsUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="communicationUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="infoServiceUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="infoProductUnit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mobilePhoneLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>移动电话用户数：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="internetLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>互联网接入数：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fixedAssetsLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>固定资产投入：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="communicationLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>通信业投入：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="infoServiceLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>信息服务收入：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="infoProductLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>信息产品收入：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="regionLabel_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>添加条目</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="regionLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>地区：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>580</y>
     <width>174</width>
     <height>36</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新增数据</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找数据</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>修改数据</string>
   </property>
  </action>
  <action name="actionStatics">
   <property name="text">
    <string>统计数据</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存到文件</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>从文件载入</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
